docs/CI.md

```
@@ -1,20 +1,491 @@
 ***
 
-Continuous integration testing is handled by GitHub Actions and the generous CircleCI service.
+Continuous integration testing for the Multicast project is handled by
+[GitHub Actions](https://github.com/reactive-firewall/multicast/actions) and the
+generous [CircleCI service](https://app.circleci.com/pipelines/github/reactive-firewall/multicast).
+
+**GitHub Actions**
+
+[GitHub Actions Metrics](https://github.com/reactive-firewall/multicast/actions/metrics/performance?dateRangeType=DATE_RANGE_TYPE_LAST_90_DAYS&tab=runner)
+
+**CircleCI**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)